ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
Did you mean "`core.js` should remain css dependency free"? You wrote `core.css`.

Always including `core.css` when any CSS is included sounds fine to me, that's an implicit dependency, but since both structure and theme files depend on `core.css`, I don't see an issue with that.